.github/workflows/github-actions.yml

```diff
@@ -15,13 +15,23 @@
 # specific language governing permissions and limitations
 # under the License.
 
-sudo: false
-language: go
-go:
-  - 1.14.x
+name: Github-Actions
 
-script:
-  - make all
+on: [push]
```

Comment:
Can you check Github Actions workflow for apache/cloudstack. We want (a) this to run on pull requests and (b) have concurrency limit (this ensure when too many commits are push on the branch or PR, it cancels older builds).